src/datetime/mod.rs

```diff
@@ -325,6 +325,14 @@ impl<Tz: TimeZone> DateTime<Tz> {
         tz.from_utc_datetime(&self.datetime)
     }
 
+    /// Fix the offset from UTC to its current value, dropping the associated timezone information.
+    /// This it useful for converting a generic `DateTime<Tz: Timezone>` to `DateTime<FixedOffset>`.
+    #[inline]
+    #[must_use]
+    pub fn fix_offset(&self) -> DateTime<FixedOffset> {
```

Comment:
Sorry to nitpick, but should this be `with_fixed_offset` to match the naming conventions?

Reply:
I don't mind. I picked `fix_offset` because it is similar to the current `Offset::fix`. It 'fixes' the offset to its current value.

I would expect that with a method with the name `with_fixed_offset` you have to supply an argument, like you have to with `with_hour`, `with_year`, etc.

Reply:
I found myself typing it that way (`fixed_offset()`) by mistake a few times.
But the method name should have a verb in it.

To continue bikeshedding: what about `to_fixed_offset`? It doesn't return a `FixedOffset`, but returning a `DateTime<FixedOffset>` does not seem too surprising to me.

Reply:
Nobody can say we didn't think about naming 😆. Changed it to `fixed_offset`.